doc/classes/Particles2D.xml

```diff
@@ -282,43 +276,62 @@
 			<description>
 			</description>
 		</method>
-	</methods>
-	<members>
+	      </methods>
+	      
+	<members>			  
 		<member name="amount" type="int" setter="set_amount" getter="get_amount">
+		  The amount of particles that will be spawned on each emission
 		</member>
 		<member name="draw_order" type="int" setter="set_draw_order" getter="get_draw_order" enum="Particles2D.DrawOrder">
+		  The particle list's drawing order 
 		</member>
 		<member name="emitting" type="bool" setter="set_emitting" getter="is_emitting">
+		  If true the emitter is currently emitting, if false it is not
 		</member>
 		<member name="explosiveness" type="float" setter="set_explosiveness_ratio" getter="get_explosiveness_ratio">
+		  The length of intervals between particle emissions, which can range from 0 to 1
 		</member>
 		<member name="fixed_fps" type="int" setter="set_fixed_fps" getter="get_fixed_fps">
+		  The particle system's targeted FPS
 		</member>
 		<member name="fract_delta" type="bool" setter="set_fractional_delta" getter="get_fractional_delta">
+		  If true the fractional delta (time between each frame) will be enabled, if false it will not
 		</member>
 		<member name="h_frames" type="int" setter="set_h_frames" getter="get_h_frames">
+		  The amount of horizontal frames
 		</member>
 		<member name="lifetime" type="float" setter="set_lifetime" getter="get_lifetime">
-		</member>
+		  The amount of seconds that each particle will be visible
+		</member>		
 		<member name="local_coords" type="bool" setter="set_use_local_coordinates" getter="get_use_local_coordinates">
+		  If true local coordinates are enabled, if false they are not
 		</member>
 		<member name="normal_map" type="Texture" setter="set_normal_map" getter="get_normal_map">
+		  2D normal map 
 		</member>
 		<member name="one_shot" type="bool" setter="set_one_shot" getter="get_one_shot">
+		  If true, the system will stop after timeout, if false it will automatically restart.
```

Comment:
Should be "If [code]true[/code]" per style guide. Second phrase after comma should be a new sentence.